generate_testdata.sh

```diff
@@ -98,7 +98,7 @@ scaffold_test_project() {
 
     $kb create api --group foo.policy --version v1 --kind HealthCheckPolicy --controller=true --resource=true --make=false
 
-    $kb create api --group apps --version v1 --kind Pod --controller=true --resource=false --make=false
+    $kb create api --group apps --version v1 --kind Deployment --controller=true --resource=false --make=false
```

Comment:
`Pod` is not part of the `apps` group, it is part of the `core` group: [link](https://github.com/kubernetes/kubernetes/blob/11a05eb9aff104c2384781c740e053907a3849e6/staging/src/k8s.io/api/core/v1/types.go#L3664).

Replaced for `Deployment` which is part of the `apps` group: [link](https://github.com/kubernetes/kubernetes/blob/11a05eb9aff104c2384781c740e053907a3849e6/staging/src/k8s.io/api/apps/v1/types.go#L254).